src/verification.rs

```diff
@@ -0,0 +1,34 @@
+use crate::{active_mock::ActiveMock, mock::Expectation};
+
+#[derive(Clone)]
+pub(crate) struct Verification {
```

Comment:
```suggestion
#[derive(Clone)]
/// A report returned by an `ActiveMock` detailing what the user expectations were and
/// how many calls were actually received since the mock was mounted on the server.
pub(crate) struct VerificationReport {
```

Reply:
done in 0c17521
I placed the comment above `derive`. Let me know if you prefer the other way.